### PR TITLE
Update mqtt.markdown

### PR DESCRIPTION
**Pull request by zraken**
When I followed the instructions it made it seem like I can configure host/port with GUI and then add the TLS options via configuration file and that did not work. When there is any GUI configuration the mqtt configuration in the yaml file is ignored. This is not obvious, hopefully the proposed addition to help documentation will be useful.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @zraken, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17227).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/mqtt.markdown [hunk 1]
```diff
@@ -39,6 +39,19 @@ To connect to your [own MQTT broker](/docs/mqtt/broker#run-your-own):
 mqtt:
   broker: IP_ADDRESS_BROKER
 ```
+Manual configuration is required when conencting to a broker over TLS.
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]:
```suggestion

Manual configuration is required when connecting to a broker over TLS.
```

File: source/_integrations/mqtt.markdown [hunk 2]
```diff
@@ -39,6 +39,19 @@ To connect to your [own MQTT broker](/docs/mqtt/broker#run-your-own):
...
+
+```yaml
+# Example configuration.yaml entry for connecting over TLS without host name verification
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]:
```suggestion
# Example configuration.yaml entry
```

File: source/_integrations/mqtt.markdown [hunk 3]
```diff
@@ -39,6 +39,19 @@ To connect to your [own MQTT broker](/docs/mqtt/broker#run-your-own):
...
+mqtt:
+  certificate: PATH_TO_CA.crt
+  broker: IP_ADDRESS_BROKER
+  port: PORT_BROKER
+  username: MQTT_USERNAME
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]:
```suggestion
  certificate: "PATH_TO_CA.crt"
  broker: "IP_ADDRESS_BROKER"
  port: 8883
  username: "MQTT_USERNAME"
```

File: source/_integrations/mqtt.markdown [hunk 4]
```diff
@@ -39,6 +39,19 @@ To connect to your [own MQTT broker](/docs/mqtt/broker#run-your-own):
...
+  password: !secret MQTT_PASSWORD
+  tls_insecure: true
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]: We should never recommend or display a copy and paste example that disables certificate verification.

```suggestion
```

File: source/_integrations/mqtt.markdown [hunk 5]
```diff
@@ -39,6 +39,19 @@ To connect to your [own MQTT broker](/docs/mqtt/broker#run-your-own):
...
+
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by zraken — COMMENTED** [review]
Accepted suggested changes.

**Review by frenck — APPROVED** [review]
Thanks, @zraken 👍

**Pull request merged**